cmd/build_worker_image.go

```diff
@@ -20,9 +20,26 @@ func buildWorkerImageCmd() *cobra.Command {
 	var b workerImageBuilder
 	cmd := &cobra.Command{
 		Use:   "build-worker-image",
-		Short: "Build an image",
+		Short: "Build a worker image (local only, no push)",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := b.build(cmd.Context()); err != nil {
+			if err := b.build(cmd.Context(), false); err != nil {
+				b.logger.Fatal(err)
+			}
+		},
+	}
+	b.addCLIFlags(cmd.Flags())
+	cmd.MarkFlagRequired("language")
+	cmd.MarkFlagRequired("version")
+	return cmd
+}
+
+func buildPushWorkerImageCmd() *cobra.Command {
```

Comment:
Multi-platform builds need to be build and pushed together; hence the new command.